code/modules/surgery/bones.dm

```diff
@@ -154,3 +156,48 @@
 	var/datum/organ/external/affected = target.get_organ(target_zone)
 	user.visible_message("<span class='warning'>[user]'s hand slips, smearing [tool] in the incision in [target]'s [affected.display_name]!</span>" , \
 	"<span class='warning'>Your hand slips, smearing [tool] in the incision in [target]'s [affected.display_name]!</span>")
+
+
+
+//////BONE MENDER/////////
+/datum/surgery_step/bone_mender
+	allowed_tools = list(
+		/obj/item/weapon/bonesetter/bone_mender = 100,
+		)
+
+	priority = 0.1 //so it trys to do this before /glue_bone
```

Comment:
>trys

Reply:
3am coding, I do appreciate the spell check help